app/views/provider/admin/dashboards/_services_filter.html.slim

```diff
@@ -1,20 +1,23 @@
-#api_filter
+- if current_user.accessible_services.size > 1
+  #api_filter
 
-javascript:
-  document.addEventListener('DOMContentLoaded', function () {
-    var allApis = #{json @services}
+    javascript:
+      document.addEventListener('DOMContentLoaded', function () {
+        var allApis = #{json @services}
 
-    var displayApis = function(filteredApis) {
-      for (api = 0; api < allApis.length; api++) {
-        document.getElementById("service_" + allApis[api].service.id).style.display = "none"
-      }
-      for (api = 0; api < filteredApis.length; api++) {
-        document.getElementById("service_" + filteredApis[api].service.id).style.display = "inline"
-      }
-    }
+        var displayApis = function(filteredApis) {
+          for (api = 0; api < allApis.length; api++) {
+            document.getElementById("service_" + allApis[api].service.id).style.display = "none"
+          }
+          for (api = 0; api < filteredApis.length; api++) {
+            document.getElementById("service_" + filteredApis[api].service.id).style.display = "inline"
+          }
+        }
 
-    window.ApiFilter({
-      apis: allApis,
-      displayApis: displayApis
-    }, 'api_filter')
-  })
+        window.ApiFilter({
+          apis: allApis,
+          displayApis: displayApis
+        }, 'api_filter')
+      })
+- else
+  h1.DashboardSection-title APIs
```

Comment:
Since it's just 1, what about _Your API_?

Reply:
No, I want to communicate that it's the header of the APIs section and that you can manage multiple api's. That's also the reason to show the 'new API' link.

Reply:
_Your APIs_ ... 
![](https://media.giphy.com/media/10IWifjzu0nOXC/giphy.gif)